skyportal/models.py

```diff
@@ -147,36 +147,24 @@ class Source(Base):
 
     def add_linked_thumbnails(self):
         sdss_thumb = Thumbnail(photometry=self.photometry[0],
-                               public_url=self.get_sdss_url(),
+                               public_url=self.sdss_url(),
                                type='sdss')
-        ps1_thumb = Thumbnail(photometry=self.photometry[0],
-                              public_url=self.get_panstarrs_url(),
-                              type='ps1')
-        DBSession().add_all([sdss_thumb, ps1_thumb])
+        dr8_thumb = Thumbnail(photometry=self.photometry[0],
+                              public_url=self.desi_dr8_url(),
+                              type='dr8')
+        DBSession().add_all([sdss_thumb, dr8_thumb])
         DBSession().commit()
 
-    def get_sdss_url(self):
+    def sdss_url(self):
```

Comment:
These still need an `@property` above.  And then removal of the `()` in all calls elsewhere.

Reply:
Sure, done.